languages/translations/ja.json

```diff
@@ -0,0 +1,493 @@
+{
+    "meta": {
+        "capitalizeFirstLetter": false
+    },
+    "v5": {
+        "constants": {
+            "ordinalize": {
+                "1": "1 つ目の",
```

Comment:
There still are whitespaces after digits, is it ok?

Reply:
Yes this is intended.  This structure should work best based on the way sentences are formed.